compiler/src/dotty/tools/dotc/ast/DesugarEnums.scala

```diff
@@ -16,21 +16,20 @@ import scala.annotation.internal.sharable
 object DesugarEnums {
   import untpd._
 
-  @sharable object CaseKind extends Enumeration {
-    val Simple, Object, Class: Value = Value
-  }
+  @sharable enum CaseKind:
```

Comment:
I don't think we should need `sharable` anymore - enums have no mutable fields I believe

Reply:
Great! I'll remove it